README.md

```diff
@@ -36,4 +36,22 @@ developed by the same group at Bell Labs that invented Unix and C.
   [twitter account](https://twitter.com/harvey_os) where we share news
   and things we are excited about.
 
+## Quickstart
+
+Bootstrap the build environment:
+
+	./bootstrap.sh
+
+Make sure to follow any additional directions bootstrap.sh may give!
+
+Now build Harvey:
+
+	ARCH=amd64 ./util/build
+
+And boot:
+
+	(HARVEY=$(pwd) && cd sys/src/9/amd64 && sh ../../../../util/GO9PRUN)
```

Comment:
you don't need the cd because GO9PRUN uses $HARVEY/sys/src/9/amd64/harvey as the name I think.
HARVEY=`pwd` sh util/GO9PRUN
should do it.